protostar/utils/data_transformer_test.py

```diff
@@ -0,0 +1,229 @@
+from typing_extensions import Literal
```

Comment:
I'd say this is an integration test because it relies on Starknet.py

Reply:
But we are not testing integration of our code swatches; instead we test single functions which uses Starknet.py as an internal dependency. 🤔

Reply:
I've decided to go with the integration tests approach, because it allows us to leverage ProtostarFixture.